boards/FSM/Src/main.c

```diff
@@ -206,63 +204,43 @@ int main(void)
 
     Io_SharedHardFaultHandler_Init();
 
-    Io_FlowMeters_Init(&htim4);
-    flow_meter_in_range_check =
-        App_InRangeCheck_Create(Io_FlowMeters_GetFlowRate, MIN_FLOW_RATE_L_PER_MIN, MAX_FLOW_RATE_L_PER_MIN);
-
-    Io_WheelSpeedSensors_Init(&htim16, &htim17);
-    left_wheel_speed_sensor_in_range_check = App_InRangeCheck_Create(
-        Io_WheelSpeedSensors_GetLeftSpeedKph, MIN_LEFT_WHEEL_SPEED_KPH, MAX_LEFT_WHEEL_SPEED_KPH);
-    right_wheel_speed_sensor_in_range_check = App_InRangeCheck_Create(
-        Io_WheelSpeedSensors_GetRightSpeedKph, MIN_RIGHT_WHEEL_SPEED_KPH, MAX_RIGHT_WHEEL_SPEED_KPH);
-
-    steering_angle_sensor_in_range_check =
-        App_InRangeCheck_Create(Io_SteeringAngleSensor_GetAngleDegree, MIN_STEERING_ANGLE_DEG, MAX_STEERING_ANGLE_DEG);
-
-    brake = App_Brake_Create(
-        Io_MSP3002K5P3N1_GetPressurePsi, Io_MSP3002K5P3N1_IsOpenOrShortCircuit, Io_Brake_IsActuated,
-        MIN_BRAKE_PRESSURE_PSI, MAX_BRAKE_PRESSURE_PSI);
-
+    //=============================================IMPORTANT CODE/=============================================
+    // Buses
     can_tx = App_CanTx_Create(
         Io_CanTx_EnqueueNonPeriodicMsg_FSM_STARTUP, Io_CanTx_EnqueueNonPeriodicMsg_FSM_WATCHDOG_TIMEOUT,
         Io_CanTx_EnqueueNonPeriodicMsg_FSM_AIR_SHUTDOWN);
-
     can_rx            = App_CanRx_Create();
     heartbeat_monitor = App_SharedHeartbeatMonitor_Create(
         Io_SharedHeartbeatMonitor_GetCurrentMs, HEARTBEAT_MONITOR_TIMEOUT_PERIOD_MS, HEARTBEAT_MONITOR_BOARDS_TO_CHECK);
 
-    rgb_led_sequence = App_SharedRgbLedSequence_Create(
-        Io_RgbLedSequence_TurnOnRedLed, Io_RgbLedSequence_TurnOnBlueLed, Io_RgbLedSequence_TurnOnGreenLed);
-
-    clock = App_SharedClock_Create();
+    // Unwrapped Ranges
+    Io_WheelSpeedSensors_Init(&htim16, &htim17);
+    wheels   = App_Wheels_Create(Io_WheelSpeedSensors_GetLeftSpeedKph, Io_WheelSpeedSensors_GetRightSpeedKph);
+    steering = App_Steering_Create(Io_SteeringAngleSensor_GetAngleDegree);
 
+    // Accelerator
     Io_PrimaryScancon2RMHF_Init(&htim1);
     Io_SecondaryScancon2RMHF_Init(&htim2);
-
     papps_and_sapps = App_AcceleratorPedals_Create(
         Io_AcceleratorPedals_IsPappsEncoderAlarmActive, Io_AcceleratorPedals_IsSappsEncoderAlarmActive,
         Io_AcceleratorPedals_GetPapps, Io_AcceleratorPedals_GetPapps);
-
     Io_PrimaryScancon2RMHF_SetEncoderCounter(PAPPS_ENCODER_UNPRESSED_VALUE);
```

Comment:
same as above, relic of quadrature

Reply:
Fixed with 14824febcf1fa04e56d4fa4c8ccd12bbf708ac58